### ISSUE TITLE
Pkg.test demo "lines_scatter.jl" throws an error/does not display

### ISSUE THREAD

**Issue by CrashBurnRepeat** [JuliaGL/GLVisualize.jl#212]
When loading `lines_scatter.jl`, the following error is displayed and the demo is skipped. 

```
LoadError: MethodError: no method matching gl_color(::Array{ColorTypes.RGBA{Float64},2})

You might have used a 2d row vector where a 1d column vector was required.
Note the difference between 1d column vector [1,2,3] and 2d row vector [1 2 3].
You can convert to a column vector with the vec() function.
Closest candidates are:
  gl_color(::Any, ::Any) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/backends/glvisualize.jl:351
  gl_color(::ColorTypes.RGBA{Float32}) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/backends/glvisualize.jl:342
  gl_color(::PlotUtils.ColorGradient) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/backends/glvisualize.jl:340
  ...
extract_c(::Dict{Symbol,Any}, ::Dict{Symbol,Any}, ::Symbol) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/backends/glvisualize.jl:532
extract_linestyle(::Dict{Symbol,Any}, ::Dict{Symbol,Any}) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/backends/glvisualize.jl:419
_display(::Plots.Plot{Plots.GLVisualizeBackend}, ::Bool) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/backends/glvisualize.jl:1091
gui(::Plots.Plot{Plots.GLVisualizeBackend}) at /Users/crashburnrepeat/.julia/v0.6/Plots/src/output.jl:134 (repeats 2 times)
include_from_node1(::String) at ./loading.jl:576
include(::String) at ./sysimg.jl:14
_test_include(::String, ::ExampleRunner.RunnerConfig) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:323
inner_test(::String, ::ExampleRunner.RunnerConfig, ::GLWindow.Screen, ::Base.RefValue{Bool}, ::Base.RefValue{Int64}, ::ExampleRunner.#increase#23{Base.RefValue{Int64}}) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:376
make_tests(::ExampleRunner.RunnerConfig) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:456
run(::ExampleRunner.RunnerConfig) at /Users/crashburnrepeat/.julia/v0.6/GLVisualize/src/../examples/ExampleRunner.jl:482
include_from_node1(::String) at ./loading.jl:576
include(::String) at ./sysimg.jl:14
include_from_node1(::String) at ./loading.jl:576
include(::String) at ./sysimg.jl:14
process_options(::Base.JLOptions) at ./client.jl:305
_start() at ./client.jl:371
while loading /Users/crashburnrepeat/.julia/v0.6/GLVisualize/examples/plots/lines_scatter.jl, in expression starting on line 35
```

System information:
Graphics/Displays:

    Intel HD Graphics 530:

      Chipset Model: Intel HD Graphics 530
      Type: GPU
      Bus: Built-In
      VRAM (Dynamic, Max): 1536 MB
      Vendor: Intel (0x8086)
      Device ID: 0x191b
      Revision ID: 0x0006
      Automatic Graphics Switching: Supported
      gMux Version: 4.0.29 [3.2.8]
      Metal: Supported

    Radeon Pro 460:

      Chipset Model: AMD Radeon Pro 460
      Type: GPU
      Bus: PCIe
      PCIe Lane Width: x8
      VRAM (Total): 4096 MB
      Vendor: ATI (0x1002)
      Device ID: 0x67ef
      Revision ID: 0x00c0
      ROM Revision: 113-C980AF-908
      Automatic Graphics Switching: Supported
      gMux Version: 4.0.29 [3.2.8]
      EFI Driver Version: 01.00.908
      Metal: Supported
      Displays:
        Color LCD:
          Display Type: Built-In Retina LCD
          Resolution: 2880 x 1800 Retina
          Pixel Depth: 32-Bit Color (ARGB8888)
          Main Display: Yes
          Mirror: Off
          Online: Yes
          Rotation: Supported
          Automatically Adjust Brightness: No
          Connection Type: DisplayPort

Hardware:

    Hardware Overview:

      Model Name: MacBook Pro
      Model Identifier: MacBookPro13,3
      Processor Name: Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Memory: 16 GB
      Boot ROM Version: MBP133.0226.B25
      SMC Version (system): 2.38f7

Memory:

    Memory Slots:

      ECC: Disabled
      Upgradeable Memory: No

        BANK 0/DIMM0:

          Size: 8 GB
          Type: LPDDR3
          Speed: 2133 MHz
          Status: OK
          Manufacturer: 0x802C
          Part Number: 0x4D5435324C31473332443450472D30393320

        BANK 1/DIMM0:

          Size: 8 GB
          Type: LPDDR3
          Speed: 2133 MHz
          Status: OK
          Manufacturer: 0x802C
          Part Number: 0x4D5435324C31473332443450472D30393320

NVMExpress:

    Apple SSD Controller:

        APPLE SSD SM1024L:

          Capacity: 1 TB (1,000,555,581,440 bytes)
          TRIM Support: Yes
          Model: APPLE SSD SM1024L
          Revision: CXS6AA0Q
          Link Width: x4
          Link Speed: 8.0 GT/s
          Detachable Drive: No
          BSD Name: disk0
          Partition Map Type: GPT (GUID Partition Table)
          Removable Media: No

Software:

    System Software Overview:

      System Version: macOS 10.12.6 (16G29)
      Kernel Version: Darwin 16.7.0
      Time since boot: 8 days 1:51

**Comment by CrashBurnRepeat** [comment]
I should have specified this occurs when running `lines_scatter.jl` through the `Pkg.test` interface.

**Comment by SimonDanisch** [comment]
Seems like it's a problem with the color attribute... Not sure what happened in Plots.jl that this doesn't work anymore. Considering that I'm working on a new version of Plots.jl (MakiE), I think I will fix this by just not specifying the colors :D

**Comment by SimonDanisch** [comment]
Fixed in: 0f5e0db remove buggy color attribute
